charts/kubewarden-controller/values.yaml

```diff
@@ -146,6 +147,10 @@ telemetry:
     # endpoint: "all-in-one-collector.jaeger.svc.cluster.local:4317"
     # tls:
     #  insecure: true
+  endpoint: ""
+  insecure: false
+  otelCollectorCertificateSecret: ""
+  otelCollectorClientCertificateSecret: ""
```

Comment:
nitpicks:

- move this block closer to the `enableOtelSidecar`
- comment this block, this is mutually exclusive to the `enableOtelSidecar`, which is enabled by default
- add a comment stating you either enable the sidecar or you have to provide all these settings
- question: should we add a comment stating what we expect to have inside of the secret? I'm referring to have a `ca.crt`, `tls.crt` and a `tls.key` keys?